Coding Guidelines.md

```diff
@@ -0,0 +1,3 @@
+Project Reunion is primarily written in C++.  It will adhere to the "CPP Core Guidelines" at https://github.com/isocpp/CppCoreGuidelines.  Although it is not anticipated that we will require extensions or modifications to these guidelines, we may create a process for such extensions or modifications in the future.  
```

Comment:
Can we link to the pretty-printed version?

```suggestion
Project Reunion is primarily written in C++.  It will adhere to the [CPP Core Guidelines](http://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines.html).  Although it is not anticipated that we will require extensions or modifications to these guidelines, we may create a process for such extensions or modifications in the future.  
```

Reply:
Missed this one, making this changes to both links.  I thought (mistakenly) that people preferred the "inline" version.